src/NzbDrone.Common.Test/ProcessProviderTests.cs

```diff
@@ -54,8 +54,9 @@ public void GetProcessById_should_return_null_for_invalid_process(int processId)
         }
 
         [Test]
-        public void Should_be_able_to_start_process()
+        public void Should_be_able_to_start_process_windows()
```

Comment:
This one fails if run in docker, but works fine in Ubuntu on a VM, we don't want to make it Windows only, but we currently ignore it on our docker test agents.